inductiva/fluids/scenarios/wind_tunnel/README.md

```diff
@@ -0,0 +1,151 @@
+- `get_force_coefficients`: Returns the force coefficients that represent the forces acting on the object. These are the drag and lift coefficients.
+
+For the pressure field, streamlines and flow slices there are easy-to-use visualizations, which have some configuration parameters. See the example below for the general overview of these parameters.
+
+The next example fetches the default post-processed files and renders the respective visualizations. The last one, fetches the full simulation output, and with the same interface the user can configure the post-processing tools to extract the metrics he desires and render the visualizations.
+
+### Example for default files:
+
+```python
+# Get the default files from WindTunnel simulation
+output = task.get_output()
+
+# Get a pressure_field mesh
+pressure_field = output.get_object_pressure_field()
+
+# Render
+pressure_field.render_frame()
+```
+
+<img src="../../../../resources/media/openfoam/default_pressure_field.png" width="400" height="300" />
+
+```python
+# Get a mesh of the streamlines 
+streamlines = output.get_streamlines()
+
+# Render the streamlines
+streamlines.render_frame(physical_field="velocity",
+                         flow_cmap="viridis",
+                         view="isometric",
+                         streamline_radius=0.1
+                         save_path="default_streamlines.png")
+```
+
+<img src="../../../../resources/media/openfoam/default_streamlines.png" width="400" height="300" />
+
+
+```python
+# Get a mesh of the flow slice
+flow_slice = output.get_flow_slice(plane="xz")
+
+# Render the pressure field over the slice
+flow_slice.render_frame(physical_field="pressure",
+                        flow_cmap="viridis",
+                        save_path="default_flow_slice.png")
+```
+
+<img src="../../../../resources/media/openfoam/default_flow_slice.png" width="400" height="300" />
+
+### Example for general Post-processing
+
+```python
+# Get all the WindTunnel simulation files
+output = task.get_output(full=True)
+
+# Get a pressure field mesh
+pressure_field = output.get_object_pressure_field()
+
+# Render
+pressure_field.render_frame(save_path="pressure_field.png")
+```
+
+<img src="../../../../resources/media/openfoam/pressure_field.png" width="400" height="300" />
+
+```python
+# Get the streamlines mesh
+streamlines = output.get_streamlines(max_time=200,
+                                     n_points=200,
+                                     initial_step_length=0.1,
+                                     source_radius=0.5,
+                                     source_center=[-3, 0, 1])
+
+# Render the streamlines
+streamlines.render_frame(physical_field="pressure",
+                         flow_cmap="viridis",
+                         view="isometric",
+                         streamline_radius=0.1
+                         save_path="streamlines.png")
```

Comment:
Aren't most of these default? Omit them, if so.

Reply:
They are, but in this way I give them a preview of the possibilities without having to explain everything at this point.